src/tests/functional/tests/bundles/base.yaml

```diff
@@ -5,29 +5,7 @@ applications:
   prometheus-libvirt-exporter:
     charm: prometheus-libvirt-exporter
     options:
-      snap_channel: stable
-  nrpe:
-    charm: nrpe
-    channel: latest/edge
-  prometheus:
-    charm: prometheus2
-    num_units: 1
-    series: focal
-  grafana:
-    charm: grafana
-    series: focal
-    channel: latest/edge # Temporary fix for 2023.10 release
-    num_units: 1
```

Comment:
I agree that we should remove the Prometheus and Grafana deployment, but should we at least deploy grafana-agent and check the relations? In grafana-agent we can run `curl localhost:12345/agent/api/v1/metrics/targets` and check status of each target.

Also I do not understand how this is  related with arm64, can you make separate PR for this?

Reply:
grafana arm64 version failed to install due to https://github.com/canonical/charm-prometheus-libvirt-exporter/actions/runs/10346737780/job/28635804387

Reply:
we can test the endpoint from the exporter endpoint itself, there's no need for grafana-agent

Reply:
It's [recommended](https://discourse.charmhub.io/t/how-to-testing-cos-integrations/14532#tests-for-the-metrics-endpoint-and-scrape-job-4) approach from o11y team, that's why I'm suggesting to do it.

Dropping lot of tests only because of arm64 it's strange approach to me. Those tests can be marked as skipped if they are run on arm64 machine, but not dropping them from normal amd64 tests.

Reply:
> It's [recommended](https://discourse.charmhub.io/t/how-to-testing-cos-integrations/14532#tests-for-the-metrics-endpoint-and-scrape-job-4) approach from o11y team, that's why I'm suggesting to do it.

That's for native COS, right? This is not native COS, this is LMA...

> Dropping lot of tests only because of arm64 it's strange approach to me. Those tests can be marked as skipped if they are run on arm64 machine, but not dropping them from normal amd64 tests.

Sounds reasonable, I can add it back. 👍🏼

Reply:
oh, wait. It's not the test failed, but the install hook failed, so we can't skip test

Reply:
> That's for native COS, right? This is not native COS, this is LMA...

Ah, I did not realize that it's LMA.... you can ignore my comment than.